cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -177,18 +177,22 @@ Example:
 			}
 
 			authGenesis := authtypes.GenesisState{}
-			clientCtx.JSONCodec.MustUnmarshalJSON(genState["auth"], &authGenesis)
-			accounts, err := authtypes.UnpackAccounts(authGenesis.Accounts)
-			if err != nil {
-				panic(err)
-			}
-			accounts = authtypes.SanitizeGenesisAccounts(accounts)
+			clientCtx.Codec.MustUnmarshalJSON(genState["auth"], &authGenesis)
```

Comment:
@ValarDragon this is questionable-- think it is right?  Basically, didn't see any difference with it enabled.